src/mock-storage-component.ts

```diff
@@ -0,0 +1,39 @@
+import { Readable } from 'stream'
+import { ContentItem, IContentStorageComponent } from './types'
+import { SimpleContentItem, streamToBuffer } from './content-item'
+
+/**
+ * @public
+ */
+export function createStorageMock(): IContentStorageComponent {
```

Comment:
Now that we're here... should we call in-memory storage? It's not really a mock...

Reply:
Totally amigo

Reply:
Changed